backends/platform/sdl/emscripten/emscripten.cpp

```diff
@@ -68,4 +89,49 @@ void OSystem_Emscripten::setFeatureState(Feature f, bool enable) {
 	}
 }
 
+Common::Path OSystem_Emscripten::getDefaultLogFileName() {
+	return Common::Path("/tmp/scummvm.log");
+}
+
+Common::Path OSystem_Emscripten::getDefaultConfigFileName() {
+	return Common::Path(Common::String::format("%s/scummvm.ini", getenv("HOME")));
+}
+
+Common::Path OSystem_Emscripten::getScreenshotsPath() {
+	return Common::Path("/tmp/");
+}
+
+Common::Path OSystem_Emscripten::getDefaultIconsPath() {
+	return Common::Path(DATA_PATH"/gui-icons/");
+}
+
+bool OSystem_Emscripten::displayLogFile() {
+	if (_logFilePath.empty())
+		return false;
+
+	exportFile(_logFilePath);
+	return true;
+}
+
+#ifdef USE_OPENGL
+OSystem_SDL::GraphicsManagerType OSystem_Emscripten::getDefaultGraphicsManager() const {
+	return GraphicsManagerOpenGL;
+}
+#endif
+
+void OSystem_Emscripten::exportFile(const Common::Path &filename) {
+	Common::File file;
+	Common::FSNode node(filename);
+	file.open(node);
+	if (!file.isOpen()) {
+		warning("Could not open file %s!", filename.toString(Common::Path::kNativeSeparator).c_str());
+		return;
+	}
+	Common::String exportName = filename.getLastComponent().toString(Common::Path::kNativeSeparator);
+	const int32 size = file.size();
+	char *bytes = new char[size + 1];
```

Comment:
Does this get deleted after use?

Reply:
no it doesn't. thanks for the feedback, I'm not used to work in languages without GC...